README.md

```diff
@@ -4,13 +4,22 @@ Async Include
 
 A simple script to asynchronously include content on an HTML page with simple markup that “just works”.
 
-    <div data-async-url="/some-url/"></div>
+    <div data-async-include-url="/some-url/"></div>
+
+Just include the `async-include.js` file on your page, activate the replace when you want like when the document is ready:
+
+```javascript
+$(document).ready(function() {
+  $('[data-async-include-url]').asyncInclude();
+});
```

Comment:
I quite like the auto-run nature of how it works, but do like having a manually runnable function too in case someone wants to manually run it at another time. Any reason to not have it still auto-run but offer the manual option too? The delay option feels way less useful without autorun.

If we wanted to go all crazy, we could do something like:

```javascript
$(document).ready(function() {
  if (!window.ASYNC_INCLUDE_NO_AUTORUN) {
    $('[data-async-include-url]').asyncInclude();
  }
});
```

but having to specify the argument feels non-DRY too. Maybe this could be abstracted as such:

```javascript
$.asyncInclude = function() {
  $('[data-async-include-url]').asyncInclude();
}

$.fn.asyncInclude = function() {
  …
}

$(function() {
  if (!$.asyncInclude.noAutorun) {
    $.asyncInclude();
  }
});
```

Also, now that we’re allowing people to manually call this function, we should probably be preventing race-conditions that can arise if the person calls it again while it’s in the middle of the AJAX call. Maybe something as simple as as the following inside the met:

```javascript
  return this.each(function() {
    var $this = $(this);

    if ($this.data('_asyncIncludeActive')) {
      return this;
    }
    $this.data('_asyncIncludeActive', true);

    …

  });
```

What do you think?